share/setup.nsi.in

```diff
@@ -59,12 +58,12 @@ CRCCheck on
 XPStyle on
 BrandingText " "
 ShowInstDetails show
-VIProductVersion ${VERSION}.@CLIENT_VERSION_BUILD@
+VIProductVersion @CLIENT_VERSION_MAJOR@.@CLIENT_VERSION_MINOR@.@CLIENT_VERSION_REVISION@.@CLIENT_VERSION_BUILD@
```

Comment:
There are other places where this is done, like
https://github.com/bitcoin/bitcoin/blob/257f750cd986641afe04316ed0b22b646b56b60b/src/bitcoin-cli-res.rc#L5

Makes sense to define this in `configure.ac`?

Reply:
Yeah, other PR. The goal of this one is really only to change the `OutFile`